### ISSUE TITLE
test html file does not include \r

### ISSUE THREAD

**Issue by thatKennedy** [bonfirefan/city-scrapers-pitt#85]
was finding some inconsistency between running my tests and running scrapy from command line.

For some reason when I ran the spider from using the scrapy command, the text from the website always began with \r. But when running the tests the html file did not include these \r.

It was easy to fix this problem - just used lstrip('\r') on the text and now it is all consistent. But it is a little strange that there is this difference.

**Comment by ben-nathanson** [comment]
@thatKennedy What was the website? I'm going to see if I can recreate the issue and probe a solution. I'm assuming `pa_utility`?

**Comment by thatKennedy** [comment]
Thanks for checking it out. It was the site for pa_utility:
http://www.puc.pa.gov/about_puc/public_meeting_calendar/public_meeting_audio_summaries_.aspx

**Comment by ben-nathanson** [comment]
I was able to recreate this issue by generating the test file from scratch and then copying over your test script and spider script. 

###  tl;dr This is a result of character encoding differences between [Windows and Unix/Linux systems](https://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html). 

I ran `scrapy shell` on that URL and found that this uses utf-8 and is getting served up on a  Windows ASP.NET server. Windows encodes new lines differently using `\r\n` whereas modern Mac OS X and Linux use just `\n`. I'm assuming you have Linux or Mac OS and when the test file was saved to your machine, it translated the Windows encodings to into Unix style. Scrapy doesn't appear to do this though as when we're using Scrapy the `\r` are still present. I didn't get a chance to look at it byte by byte, but this seems likely to be the cause for why Scrapy and Pytest are telling different stories. 

Here's what it looked like in Scrapy Shell
```html
<article id="content">\r\n\t\t
    <h1>\r\n\tPublic Meeting Agendas\xa0</h1>\r\n
    <p>\r\n\t<a href="/about_puc/pm_videos.aspx" target="_blank">Public Meeting videos</a> are now available on the website.\xa0Video clips of a public meeting are not considered official transcripts. Official transcripts are available from the court reporting service.</p>\r\n
    <p>\r\n\tTo watch the Public Meetings live, you can watch the <a href="/about_puc/live_streaming_video.aspx" target="blank">streaming video</a> of Public Meeting.</p>\r\n
...
```

And that same blurb as it appears in the test file:
```html
<article id='content'>
		        <h1>
	Public Meeting Agendas&nbsp;</h1>
<p>
	<a href="/about_puc/pm_videos.aspx" target="_blank">Public Meeting videos</a> are now available on the website.&nbsp;Video clips of a public meeting are not considered official transcripts. Official transcripts are available from the court reporting service.</p>
<p>
	To watch the Public Meetings live, you can watch the <a href="/about_puc/live_streaming_video.aspx" target="blank">streaming video</a> of Public Meeting.</p>
...
``` 
I'm going to close this since we can't write our own operating system, but I will add this in the new docs. 

Thanks!

Ben
